addons/fortify/XEH_postInit.sqf

```diff
@@ -63,3 +66,21 @@ GVAR(objectRotationZ) = 0;
         [_object, 0, ["ACE_MainActions"], _removeAction] call ACEFUNC(interact_menu,addActionToObject);
     };
 }] call CBA_fnc_addEventHandler;
+
+[QGVAR(setMarkerVisible), {
+    params ["_object"];
+    [
+        {!isNull player},
+        {
+            params ["_object"];
+            private _objectSide = _object getVariable QGVAR(objectSide);
+            private _playerSide = side group player;
```

Comment:
What happens if the player changes side mid-mission or remote controls a unit?

Reply:
TeamSwitch EH + CBA_fnc_addPlayerEventHandler covers player side change or RC, I can add that.

I do not believe there is a way to detect side relation changes (i.e setFriend) without a constant check.

Reply:
Which `CBA_fnc_addPlayerEventHandler`? I do not see one added by this PR. Also, there is a CBA player event handler for `"group"`.

Reply:
Typo, I meant to say "could cover". I will be implementing those 2 player EHs to recalculate the desired alphas when the current unit/group changes.

I've done some digging and I still don't see a way to handle side relation changes.